content/testing-farm/_index.md

```diff
@@ -8,107 +8,145 @@ weight: 4
 Testing Farm is Packit's testing system.
 It is a containerized service for running tests.
 Test execution is configured via [Flexible Metadata Format][fmf],
-according to the [L1 Metadata][l1] and [L2 Metadata][l2] specification.
+according to the [Metadata Specification][spec].
 
-Currently only a subset of the metadata specification is implemented:
-It is possible to run [BeakerLib](#beakerlib-tests) tests and arbitrary [shell](#shell-tests) commands.
-See [Examples](#examples) and [Support Matrix](#support-matrix) for more details.
-
-If you are interested in Testing Farm's architecture, see the section [Architecture](#architecture).
+Go through the [User Guide](#user-guide) to get a quick start introduction.
+See the [Examples](#examples) section to get some more inspiration.
+The [Support Matrix](#support-matrix) gives overview of what's currently supported.
+In the [Architecture](#architecture) section you will find information about Testing Farm's internals.
 
 ## User Guide
 
-### Add Tests
+The easiest way to get started is to use the [tmt][tmt] tool which will help you with the setup.
+Install the latest stable version directly from Fedora. See [docs][tmt] for more installation options.
 
-Test configuration is stored directly in the project repository.
-Make sure that you have `fmf` package [installed][install] on your system.
-In order to start, initialize the metadata tree in the root directory of your git repository.
+    sudo dnf install -y tmt
 
-    fmf init
+### Enable Testing
 
-The initialization will create `.fmf` directory in your repository which you **must** add in your pull request otherwise Testing Farm will not find your tests.
+In order to enable test execution simply include `copr_build` and `tests` jobs in the `.packit.yaml` configuration:
 
-For small projects store test config in a single `ci.fmf` file.
-If your config grows consider using [elasticity][elasticity] to organize your config into multiple files.
-Here's a minimal example showing how to enable a simple smoke test:
+    jobs:
+    - job: copr_build
+      trigger: pull_request
+      metadata:
+        targets:
+        - fedora-all
```

Comment:
Since people usually copy the example as is. Do they want to test also for rawhide? (We may use a `fedora-stable` in all examples.)

What do you think?

Reply:
Below it mentions you can also use other targets, I believe this is ok :)